File: ZeroJudge/Normal/C++/a/a149.cpp
/*
 Problem link: https://zerojudge.tw/ShowProblem?problemid=a149
 Status: AC
*/

#include <iostream>
#include <cstdlib>

using namespace std;

int main(void) {
	int T;

	cin >> T;

	for (int i = 0; i < T; i++) {
		int temp;
        cin >> temp;

        if (temp == 0) {
            cout << 0 << endl;
        } else {
            int rest = 0;
		    int sum = 1;
		
		    while (temp > 0) {
			    rest = temp % 10;
			    temp /= 10;
			    sum *= rest;
		    }

		    cout << sum << endl;
        }
	}

	return 0;
}

File: 2023/ZeroJudge/a/a149/a149.py
"""
Created Date: 2023-11-17
Author: @1chooo (Hugo ChunHo Lin)
Problem link: https://zerojudge.tw/ShowProblem?problemid=a149
Status: AC
"""

n = int(input())

for i in range(n):
    m = int(input())

    if m == 0:
        ans = 0
    else:
        ans = 1
        while m:
            ans *= m % 10
            m //= 10
    
    print(ans)
